data/en/census_individual_gb_wls.json

```diff
@@ -0,0 +1,10762 @@
+{
```

Comment:
This schema will be removed before squash, only using for quick launch purposes.